Tests/SonarScanner.MSBuild.Tasks.IntegrationTest/TargetsTests/RoslynTargetsTests.cs

```diff
@@ -125,13 +125,13 @@ public void Settings_ValidSetup_LegacyServer_Override_Analyzers()
 
   <ItemGroup>
     <!-- all analyzers specified in the project file should be removed -->
-    <Analyzer Include='c:\should.be.removed.analyzer2.dll' />
-    <Analyzer Include='should.be.removed.analyzer1.dll' />
+    <Analyzer Include='c:\should.be.preserved.analyzer2.dll' />
+    <Analyzer Include='should.be.preserved.analyzer1.dll' />
   </ItemGroup>
   <ItemGroup>
     <!-- These additional files don't match ones in the config and should be preserved -->
-    <AdditionalFiles Include='should.not.be.removed.additional1.txt' />
-    <AdditionalFiles Include='should.not.be.removed.additional2.txt' />
+    <AdditionalFiles Include='should.be.preserved.additional1.txt' />
+    <AdditionalFiles Include='should.be.preserved.additional2.txt' />
 
     <!-- This additional file matches one in the config and should be replaced -->
```

Comment:
they're actually "removed" rather than "replaced", right?

if so, could you update all comments? 

The anayzer comment at line 127 is ok. The additional files comments seem to be wrong.